protocol/contracts/beanstalk/init/replant/Replant1.sol

```diff
@@ -82,7 +82,7 @@ contract Replant1 {
 
     function init() external {
         // 1. Remove Deposits
-        LibTokenSilo.removeDeposit(
+        LibTokenSilo.removeDepositFromAccount(
```

Comment:
Do we need to keep these one time use legacy files up to date with changes? Seems best to keep them in the state that they were used in.

Reply:
@silochad, do you have an opinion?